understanding/dragging-movements.html

```diff
@@ -56,213 +56,154 @@
+                  <p>この達成基準は、<a href="pointer-gestures">達成基準 2.5.1 ポインタジェスチャ</a>で説明されているポインタジェスチャではなく、ドラッグ動作に適用される。ポインタジェスチャには、方向性のある軌跡ベースのジェスチャだけでなくマルチポイントのジェスチャも含まれる。対照的に、ドラッグ動作の場合は、実際の軌跡ではなく、動作の開始点と終了点だけが問題になる。
                   </p>
                   
                   
-                  <p>Additional examples are selection rectangles that set the first x/y rectangle coordinate
-                     at the pointer position via a pointer down-event, and the second x/y coordinate, after
-                     a dragging movement, at the next up-event. A similar example is a connecting line
-                     drawn between two different items on the screen, as in an allocation test where users
-                     are required to draw a line between questions and corresponding answers. In these
-                     cases, the dragging movement requires an alternative way to accomplish the same action
-                     that does not rely on the dragging movement. For example, two separate single tap
-                     or click actions may define the rectangle coordinates or the start and end points
-                     of a connecting line.
+                  <p>追加の例としては、ポインタのダウンイベントによってそのポインタの位置に 1 番目の x/y 直交座標を設定し、ドラッグ移動後の次のアップイベントで 2 番目の x/y 座標を設定する選択範囲がある。同様の例は、利用者が質問と対応する回答の間に線を引くように要求される割り当てテストのように、画面上の二つの異なる項目の間に引かれる接続線である。このような場合、ドラッグ動作には、ドラッグ動作に依存しない同じアクションを達成する別の方法が必要である。例えば、二つの別々のシングルタップ又はクリック操作で、直交座標、又は接続線の始点及び終点を定義できる。
                   </p>
                   
                   
                </section>
                <section id="alternatives-for-dragging-movements-on-the-same-page">
                   
-                  <h3>Alternatives for dragging movements on the same page</h3>
+                  <h3>同一ページ上でのドラッグ動作の代替</h3>
                   
                   
-                  <p>Where functionality can be executed via dragging movements and an equivalent option
-                     exists that allows for single-pointer access without dragging, this Success Criterion
-                     is passed. It does not have to be the same component, so long as the functionality
-                     is equivalent. An example is a color wheel where a color can be changed by dragging
-                     an indicator. In addition, text fields for the numerical input of color values allow
-                     the definition of a color without requiring dragging movements. (Note that a text
-                     input is considered device agnostic; although the purpose is to enter characters,
-                     text entry can take place through voice, pointer or keyboard.)
+                  <p>ドラッグ操作によって機能を実行できる、及びドラッグせずにシングルポインタアクセスを可能にする同等のオプションが存在する場合、この達成基準に合格する。機能が同等である限り、同じコンポーネントである必要はない。例としては、インジケーターをドラッグすることで色を変更できる色相環がある。加えて、色の値を数値入力するテキストフィールドは、ドラッグ操作を必要とせずに色の定義を可能にする。(テキスト入力はデバイスに依存しないと見なされることに注意する。目的は文字を記入することであるが、テキスト入力は音声、ポインタ、又はキーボードを通じて行われることがある。)
```

Comment:
原文では「dragging movements」なので、「操作」を「動作」にしてみました。

```suggestion
                  <p>ドラッグ動作によって機能を実行できる、及びドラッグせずにシングルポインタアクセスを可能にする同等のオプションが存在する場合、この達成基準に合格する。機能が同等である限り、同じコンポーネントである必要はない。例としては、インジケーターをドラッグすることで色を変更できる色相環がある。加えて、色の値を数値入力するテキストフィールドは、ドラッグ動作を必要とせずに色の定義を可能にする。(テキスト入力はデバイスに依存しないと見なされることに注意する。目的は文字を記入することであるが、テキスト入力は音声、ポインタ、又はキーボードを通じて行われることがある。)
```